calendar-adjustment/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Calendar adjustment for monthly time series",
+  "description": "Given a dataset containing monthly time series data with datestamps, normalizes the time series values by the number of days per month (with leap year detection). Suitable for monthly aggregated data (e.g. total liters per month).",
```

Comment:
adding here a heads-up to @osroca, who will tell us if this description is too long to show nicely in the UI...